### ISSUE TITLE
Any chance you could help me figure out how to interface with the Sharp Wizard?

### ISSUE THREAD

**Issue by techopswhiz** [arpruss/ozdev#1]
I have an old sharp wizard lying around. Mine had a game on it called dungeon which I'd like to get the file for and maybe even it's code if possible.

Do you have anything that can help me?

Software, manuals, or just knowledge of how it works?  I think I have the cable somewhere, but I'll have to get a converter to usb.  But past that, is there software that works with it now?

Thanks!

**Comment by arpruss** [comment]
The downloader is here: http://www.sharp-world.com/contents/organizer/software/zq-700/index.html
You should be able to upload .wzd files from this project to the device here.
And the project here should allow development, except that you may need to run some of the .exe files in a 32-bit virtual machine.

**Comment by techopswhiz** [comment]
I think mine is an OZ 750 - is that compatible? Were your programs for ZQs?

**Comment by arpruss** [comment]
Yeah, they are all pretty much the same. The OZ 750 and ZQ 750 are the same, except for labeling. The OZ 770 is a slightly upgraded model, and the OZ 730 is a slightly downgraded one.
